tests/ciAutoload.php

```diff
@@ -0,0 +1,81 @@
+<?php
```

Comment:
This file assumes that everything in symbini.php is defined. We should probably add defaults if values are missing.